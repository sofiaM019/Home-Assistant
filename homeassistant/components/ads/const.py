"""Support for Automation Device Specification (ADS)."""

from __future__ import annotations

from enum import StrEnum
from typing import TYPE_CHECKING

from homeassistant.util.hass_dict import HassKey

if TYPE_CHECKING:
    from .hub import AdsHub

DOMAIN = "ads"

DATA_ADS: HassKey[AdsHub] = HassKey(DOMAIN)
STATE_KEY_STATE = "state"

SERVICE_WRITE_DATA_BY_NAME = "write_data_by_name"

CONF_ADS_VAR = "adsvar"
CONF_ADS_FACTOR = "factor"
CONF_ADS_TYPE = "adstype"
CONF_ADS_VALUE = "value"


class AdsType(StrEnum):
    """Supported Types."""

    BOOL = "bool"
    BYTE = "byte"
    INT = "int"
    UINT = "uint"
    SINT = "sint"
    USINT = "usint"
    DINT = "dint"
    UDINT = "udint"
    WORD = "word"
    DWORD = "dword"
    LREAL = "lreal"
    REAL = "real"
    STRING = "string"
    TIME = "time"
    DATE = "date"
    DATE_AND_TIME = "dt"
    TOD = "tod"
