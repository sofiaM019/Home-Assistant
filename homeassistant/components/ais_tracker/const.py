"""AIS tracker constants."""

DOMAIN = "ais_tracker"

CONF_MMSIS = "mmsis"
DEFAULT_PORT = 12346
