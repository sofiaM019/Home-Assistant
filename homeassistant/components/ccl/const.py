"""Constants for the CCL Electronics integration."""

DOMAIN = "ccl"
