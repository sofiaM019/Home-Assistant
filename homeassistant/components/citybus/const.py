"""CityBus constants"""

DOMAIN = "citybus"

CONF_ROUTE = "route"
CONF_DIRECTION = "direction"
CONF_STOP = "stop"
