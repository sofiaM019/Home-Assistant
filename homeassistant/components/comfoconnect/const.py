"""Constants for the ComfoConnect integration."""

DOMAIN = "comfoconnect"

CONF_USER_AGENT = "user_agent"
DEFAULT_PIN = 0
DEFAULT_TOKEN = "00000000000000000000000000000001"
DEFAULT_USER_AGENT = "Home Assistant"
