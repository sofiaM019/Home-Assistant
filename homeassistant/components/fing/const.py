"""Const for the Fing integration."""

AGENT_IP = "agent_ip"
AGENT_PORT = "agent_port"
AGENT_KEY = "agent_key"
DOMAIN = "fing"
