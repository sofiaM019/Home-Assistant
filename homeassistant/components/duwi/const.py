"""Constants for the DUWI integration."""

from enum import StrEnum
import logging

from homeassistant.const import Platform

_LOGGER = logging.getLogger(__package__)

MANUFACTURER = "duwi"

DOMAIN = "duwi"

APP_VERSION = "0.1.1"

CLIENT_VERSION = "0.1.1"

CLIENT_MODEL = "homeassistant"

DUWI_DISCOVERY_NEW = "duwi_discovery_new"
DUWI_SCENE_UPDATE = "duwi_scene_update"
DUWI_HA_SIGNAL_UPDATE_ENTITY = "duwi_entry_update"
DUWI_HA_ACCESS_TOKEN = "duwi_access_token"

# API keys
CLIENT = "client"
ADDRESS = "address"
WS_ADDRESS = "ws_address"
APP_KEY = "app_key"
APP_SECRET = "app_secret"
ACCESS_TOKEN = "access_token"
REFRESH_TOKEN = "refresh_token"
PHONE = "phone"
PASSWORD = "password"
HOUSE_NO = "house_no"
HOUSE_NAME = "house_name"
HOUSE_KEY = "house_key"
CONF_TOKEN_INFO = "token_info"

# List of platforms that support config entry
SUPPORTED_PLATFORMS = [
    Platform.SWITCH,
]


class DPCode(StrEnum):
    """Data Point Codes used by Duwi."""

    # button
    BUTTON = "button"

    # switch
    SWITCH = "switch"

    # light
    ON_OFF = "on_off"
    LIGHT = "light"
    COLOR_TEMP = "color_temp"
    COLOR = "color"
    RGB = "rgb"
    RGBW = "rgbw"
    RGBCW = "rgbcw"

    # cover
    ROLLER = "roller"
    SHUTTER = "shutter"
    SHUTTER_2 = "shutter-2"

    # music
    HUA_ERSI_MUSIC = "hua_ersi_music"
    XIANG_WANG_MUSIC_S7_MINI_3S = "xiangwang_music_s7_mini_3s"
    XIANG_WANG_MUSIC_S8 = "xiangwang_music_s8"
    SHENG_BI_KE_MUSIC = "sheng_bike_music"
    BO_SHENG_MUSIC = "bo_sheng_music"
    YOU_DA_MUSIC = "youda_music"

    # sensor
    TEMP = "temp"
    HUMIDITY = "humidity"
    HCHO = "hcho"
    PM25 = "pm25"
    CO2 = "co2"
    BRIGHT = "bright"
    IQA = "iaq"
    HUMAN = "human"
    TRIGGER = "trigger"
    MOISTURE = "moisture"
    SMOKE = "smoke"
    MOVING = "moving"
    GAS = "gas"
    DOOR = "door"
    OPENING = "opening"
    CO = "co"
    TVOC = "tvoc"
    PM10 = "pm10"

    # climate
    AC = "ac"
    FA = "fa"
    FH = "fh"
    HP = "hp"
    TC = "tc"
