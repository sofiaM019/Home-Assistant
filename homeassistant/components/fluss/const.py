"""Constants for the Fluss+ integration."""

from homeassistant.const import CONF_API_KEY  # noqa: F401

DOMAIN = "fluss"
BASE_URL = "https://zgekzokxrl.execute-api.eu-west-1.amazonaws.com/v1/api/"

CONF_BASE_URL = BASE_URL  # You can refer to BASE_URL this way
