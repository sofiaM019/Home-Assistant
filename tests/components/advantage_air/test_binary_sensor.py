"""Test the Advantage Air Binary Sensor Platform."""

from unittest.mock import AsyncMock

import pytest
from syrupy.assertion import SnapshotAssertion

from homeassistant.const import Platform
from homeassistant.core import HomeAssistant
from homeassistant.helpers import entity_registry as er

from . import add_mock_config, assert_entities


@pytest.mark.usefixtures("entity_registry_enabled_by_default")
async def test_binary_sensor(
    hass: HomeAssistant,
    entity_registry: er.EntityRegistry,
    mock_get: AsyncMock,
    snapshot: SnapshotAssertion,
) -> None:
    """Test binary sensor setup."""

    entry = await add_mock_config(hass, [Platform.BINARY_SENSOR])
    assert_entities(hass, entry.entry_id, entity_registry, snapshot)
