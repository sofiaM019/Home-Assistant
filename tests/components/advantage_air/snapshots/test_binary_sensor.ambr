# serializer version: 1
# name: test_binary_sensor[binary_sensor.myauto_filter-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.myauto_filter',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.PROBLEM: 'problem'>,
    'original_icon': None,
    'original_name': 'Filter',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac3-filter',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myauto_filter-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'problem',
      'friendly_name': 'myauto Filter',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myauto_filter',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_filter-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.mytemp_filter',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.PROBLEM: 'problem'>,
    'original_icon': None,
    'original_name': 'Filter',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac2-filter',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_filter-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'problem',
      'friendly_name': 'mytemp Filter',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.mytemp_filter',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_a_motion-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': None,
    'entity_id': 'binary_sensor.mytemp_zone_a_motion',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.MOTION: 'motion'>,
    'original_icon': None,
    'original_name': 'Zone A motion',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac2-z01-motion',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_a_motion-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'motion',
      'friendly_name': 'mytemp Zone A motion',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.mytemp_zone_a_motion',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_a_myzone-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.mytemp_zone_a_myzone',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': None,
    'original_icon': None,
    'original_name': 'Zone A myZone',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac2-z01-myzone',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_a_myzone-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'mytemp Zone A myZone',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.mytemp_zone_a_myzone',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_b_motion-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': None,
    'entity_id': 'binary_sensor.mytemp_zone_b_motion',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.MOTION: 'motion'>,
    'original_icon': None,
    'original_name': 'Zone B motion',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac2-z02-motion',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_b_motion-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'motion',
      'friendly_name': 'mytemp Zone B motion',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.mytemp_zone_b_motion',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_b_myzone-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.mytemp_zone_b_myzone',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': None,
    'original_icon': None,
    'original_name': 'Zone B myZone',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac2-z02-myzone',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.mytemp_zone_b_myzone-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'mytemp Zone B myZone',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.mytemp_zone_b_myzone',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_filter-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.myzone_filter',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.PROBLEM: 'problem'>,
    'original_icon': None,
    'original_name': 'Filter',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-filter',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_filter-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'problem',
      'friendly_name': 'myzone Filter',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_filter',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_3_motion-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': None,
    'entity_id': 'binary_sensor.myzone_zone_3_motion',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.MOTION: 'motion'>,
    'original_icon': None,
    'original_name': 'Zone 3 motion',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z03-motion',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_3_motion-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'motion',
      'friendly_name': 'myzone Zone 3 motion',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_3_motion',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_3_myzone-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.myzone_zone_3_myzone',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': None,
    'original_icon': None,
    'original_name': 'Zone 3 myZone',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z03-myzone',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_3_myzone-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'myzone Zone 3 myZone',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_3_myzone',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_4_myzone-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.myzone_zone_4_myzone',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': None,
    'original_icon': None,
    'original_name': 'Zone 4 myZone',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z04-myzone',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_4_myzone-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'myzone Zone 4 myZone',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_4_myzone',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_5_myzone-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.myzone_zone_5_myzone',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': None,
    'original_icon': None,
    'original_name': 'Zone 5 myZone',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z05-myzone',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_5_myzone-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'myzone Zone 5 myZone',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_5_myzone',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_closed_with_sensor_motion-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': None,
    'entity_id': 'binary_sensor.myzone_zone_closed_with_sensor_motion',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.MOTION: 'motion'>,
    'original_icon': None,
    'original_name': 'Zone closed with Sensor motion',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z02-motion',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_closed_with_sensor_motion-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'motion',
      'friendly_name': 'myzone Zone closed with Sensor motion',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_closed_with_sensor_motion',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_closed_with_sensor_myzone-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.myzone_zone_closed_with_sensor_myzone',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': None,
    'original_icon': None,
    'original_name': 'Zone closed with Sensor myZone',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z02-myzone',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_closed_with_sensor_myzone-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'myzone Zone closed with Sensor myZone',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_closed_with_sensor_myzone',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'off',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_open_with_sensor_motion-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': None,
    'entity_id': 'binary_sensor.myzone_zone_open_with_sensor_motion',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': <BinarySensorDeviceClass.MOTION: 'motion'>,
    'original_icon': None,
    'original_name': 'Zone open with Sensor motion',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z01-motion',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_open_with_sensor_motion-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'device_class': 'motion',
      'friendly_name': 'myzone Zone open with Sensor motion',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_open_with_sensor_motion',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_open_with_sensor_myzone-entry]
  EntityRegistryEntrySnapshot({
    'aliases': set({
    }),
    'area_id': None,
    'capabilities': None,
    'config_entry_id': <ANY>,
    'device_class': None,
    'device_id': <ANY>,
    'disabled_by': None,
    'domain': 'binary_sensor',
    'entity_category': <EntityCategory.DIAGNOSTIC: 'diagnostic'>,
    'entity_id': 'binary_sensor.myzone_zone_open_with_sensor_myzone',
    'has_entity_name': True,
    'hidden_by': None,
    'icon': None,
    'id': <ANY>,
    'labels': set({
    }),
    'name': None,
    'options': dict({
    }),
    'original_device_class': None,
    'original_icon': None,
    'original_name': 'Zone open with Sensor myZone',
    'platform': 'advantage_air',
    'previous_unique_id': None,
    'supported_features': 0,
    'translation_key': None,
    'unique_id': 'uniqueid-ac1-z01-myzone',
    'unit_of_measurement': None,
  })
# ---
# name: test_binary_sensor[binary_sensor.myzone_zone_open_with_sensor_myzone-state]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'myzone Zone open with Sensor myZone',
    }),
    'context': <ANY>,
    'entity_id': 'binary_sensor.myzone_zone_open_with_sensor_myzone',
    'last_changed': <ANY>,
    'last_reported': <ANY>,
    'last_updated': <ANY>,
    'state': 'on',
  })
# ---
